run_page/garmin_sync_cn_global.py

```diff
@@ -83,3 +94,31 @@
         garmin_global_client.upload_activities_files(to_upload_files, False)
     )
     loop.run_until_complete(future)
+
+    # Save synced activity list for speeding up
+    synced_activity.extend(new_ids)
+    save_synced_activity_list(synced_activity)
+
+    # Step 2:
+    # Generate track from gpx file
+
+    # Todo: use fit file to directly generate track after fixing issue 584
```

Comment:
I believe we should prioritize addressing issue #584 before proceeding with this PR since the code will require fixing later on.

Reply:
I agree